### DISCUSSION TITLE
Osu!Lazer Droid on osu mania and performance issues

### DISCUSSION THREAD

**Discussion by netflexs** [ppy/osu#25107] [Ideas]
can you make an osu mania ui portrait cause when played on landscape its lit hard to play cause the scale of ui is so tiny in my device

and for a performance issue my device just get an 14 fps on 4gb ram and poor cpu like mediatek p35

just that,
Thank You!

**Comment by peppy** [comment]
https://github.com/ppy/osu/issues/23377
